ChangeLog.md

```diff
@@ -25,6 +25,15 @@ See docs/process.md for more on how version tagging works.
   ports of native GL renderers from later accidentally attempting to activate
   "dormant" features if web browser implementations gain new WebGL extensions in
   the future, which `*glGetProcAddress()` is not able to support. (#20802)
+- Added Hi DPI support to GLFW. When enabled, GLFW automatically accounts for the
+  `devicePixelRatio` browser property and changes the size of the canvas accordingly
+  (including dynamically if the canvas is moved from a 4k screen to a 2k screen and 
+  vice-versa). `glfwGetFramebufferSize` now properly returns the canvas size in pixels, 
+  while `glfwGetWindowSize` returns the canvas size is screen size. By default,
+  this feature is disabled. You can enable it before creating a window by calling 
+  `glfwWindowHint(GLFW_SCALE_TO_MONITOR, GLFW_TRUE)`. You can also 
+  dynamically change it after the window has been created by calling
+  `glfwSetWindowAttrib(window, GLFW_SCALE_TO_MONITOR, GLFW_TRUE)`.
```

Comment:
Can you add the PR number in brackets after the text?

Reply:
Done.